packages/react-dom/src/client/ReactDOM.js

```diff
@@ -736,7 +737,7 @@ const DOMRenderer = ReactFiberReconciler({
     },
 
     resetTextContent(domElement: Instance): void {
-      domElement.textContent = '';
+      setTextContent(domElement, '');
```

Comment:
Semi related:
https://github.com/facebook/react/issues/11609

 `commitTextUpdate` throws in some cases in IE9. I wonder if using this method there would fix it.